app/views/donations/new.html.erb

```diff
@@ -28,23 +28,31 @@
       </div>
     </div>
     <div class="row">
-      <div class="small-12 medium-offset-2 medium-4 columns">
-        <div class="row">
-          <div class="small-6 columns">
-            <%= f.file_field :picture, required: "" %>
+      <div class="small-12 text-center columns">
+        <span>Upload a picture of your donation!</span>
+      </div>
+    </div>
+    <div class="row">
+      <div class="small-offset-5 small-1 columns">
+        <label class="text-center donation_picture_button columns">
+            <input id="donation_picture" name="donation[picture]" required="required" type="file" data-invalid="" aria-invalid="true" onchange="showFilePath()"/>
+            <span>Upload</span>
             <small class="error">Picture is required.</small>
-          </div>
-          <div class="small-6 columns">
-            <span>Upload a picture of your donation!</span>
-          </div>
-        </div>
+        </label>  
       </div>
-      <div class="small-12 medium-6 end columns">
-        <div class="row">
-            <div class="small-12 end columns">
-                <%= f.check_box :refrigeration %> <label for="donation_refrigeration">Does your donation require refrigeration?</label>
-          </div>
-        </div>
+      <div class="no-file-selected small-6 columns">
+        <span id="no-file-selected">No file selected</span>
+      </div>
+      <script>
+        function showFilePath() {
+            var x=document.getElementById("donation_picture");
+            document.getElementById('no-file-selected').innerHTML=x.value;
```

Comment:
One last small complaint: when I upload a picture now, it says "C:\fakepath\Screen Shot 2015-04-30 at 16.09.45.png".  The fakepath thing is because browsers don't run javascript that can access your local filesystem (which makes sense).  However, can you just add a short regex or something to make sure we just get the filename.  See [this SO post](http://stackoverflow.com/questions/857618/javascript-how-to-extract-filename-from-a-file-input-control) which can probably just do it for you.